exchanges/itbit/itbit_wrapper.go

```diff
@@ -305,8 +306,32 @@ func (i *ItBit) GetFundingHistory() ([]exchange.FundHistory, error) {
 }
 
 // GetExchangeHistory returns historic trade data since exchange opening.
-func (i *ItBit) GetExchangeHistory(p currency.Pair, assetType asset.Item) ([]exchange.TradeHistory, error) {
-	return nil, common.ErrNotYetImplemented
+func (i *ItBit) GetExchangeHistory(req *trade.HistoryRequest) ([]trade.History, error) {
+	var resp []trade.History
+
+	trades, err := i.GetTradeHistory(i.FormatExchangeCurrency(req.Pair, req.Asset).String(),
```

Comment:
![image](https://user-images.githubusercontent.com/9261323/81357351-bdedae80-9116-11ea-8876-3b329564bf9d.png)
Need to make tradeID optional if not present